hazelcast/src/test/java/com/hazelcast/map/LocalMapStatsTest.java

```diff
@@ -162,6 +162,17 @@ public void run()
         });
     }
 
+    @Test
+    public void testDelete() throws Exception {
+        IMap<Integer, Integer> map = getMap();
+        for (int i = 0; i < 100; i++) {
+            map.put(i, i);
+            map.delete(i);
+        }
+        LocalMapStats localMapStats = geMapStats();
+        assertEquals(100, localMapStats.getRemoveOperationCount());
```

Comment:
can you also add the assert for getTotalRemoveLatency being grater than zero?

Reply:
As you will remember, the latency could report 0 because it could be too short and System.nanotime resolution sometimes too big.  We don't have this check in any other similar tests on this class. So I am skipping it.